poker-pals/src/components/Poker.vue

```diff
@@ -73,18 +72,11 @@ export default {
         TableLayout,
         Display,
     },
-    created() {
-        this.socket = io("http://localhost:3000");
```

Comment:
Why are you removing this? You're still using sockets in this file

Reply:
line 69 of tables.vue passes its own socket. this is needed as when the user selects a table in tables.vue i need their socket to remain the same else the tables.vue socket gets added to the room and the poker.vue socket is left out.